battlepass/src/lib.rs

```diff
@@ -444,10 +452,10 @@ pub mod pallet {
 			ensure!(Self::check_battlepass_state(battlepass_id, BattlepassState::ACTIVE)?, Error::<T>::BattlepassStateWrong);
 			// check if Org is active
 			ensure!(T::Control::is_org_active(&battlepass.org_id), Error::<T>::OrgUnknownOrInactive);
-			// check permissions (prime, bot, self - temp)
-			ensure!(by_who == for_who || Self::is_prime(&battlepass.org_id, by_who.clone())?, Error::<T>::AuthorizationError);
 			// check if user is a member of organization
 			ensure!(T::Control::is_org_member_active(&battlepass.org_id, &for_who), Error::<T>::NotMember);
+			// check permissions (self, prime)
+			ensure!(by_who == for_who || Self::is_prime(&battlepass.org_id, by_who.clone())?, Error::<T>::AuthorizationError);
```

Comment:
Bot should be able to claim it for the user as well.

Reply:
Updated.